### PR TITLE
Commands to create rules and notifications.

### PR DESCRIPTION
**Pull request by sersut**
@jamesc 

This can be useful if you'd like to automate creation of rules and notifications. For e.g. you are preparing a demo for an important event, say ChefConf, and you would like to make sure you can automate creation of these :smile: 

I think we should version bump and release if these look good.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,13 +16,48 @@ knife audit list
...
+
+## Example data for creating rules and notifications:
+
+### Rules
+```
+{
+  "name": "New Rule Group 1",
+  "modified_by": "admin",
+  "with": {
+    "priority": 0
```
Comment (bookshelfdave) [inline README.md:33]: it's weird having the priority here as well as in the rule. See https://github.com/chef/alaska_rules#rule-group-options
I can't remember why we do this, something to do with the front end.

**Comment by bookshelfdave** [comment]
very nice!

File: README.md (hunk 1 above)
Comment (sersut) [inline README.md:33]: Shall I remove this from here? I've added it because the UI was sending it.

**Comment by jamesc** [comment]
Nice!

I'd like it if we could have a rules block in a file rather than the JSON (and didn't have a clean way to do it, so I didn't do it!).

so we'd do:

```
knife rules create --priority 1 --active false --name "these rules rock"  rules.json
```

File: README.md (hunk 1 above)
Comment (bookshelfdave) [inline README.md:33]: I believe notifier_config needs it to sort: https://github.com/chef/notifier_config/blob/master/src/nc_obj_rule.erl

I wonder if a regex can pull it out so a user doesn't have to enter it in the rule and as a param?

**Pull request merged**